tasks.py

```diff
@@ -281,6 +281,14 @@ def help(c):
     except:
         pass
 
+@task
+def generate_certificates(c):
+    """
+    Generates the cert.pem and cert-key.pem files
+    """
+    with c.cd(c.project_directory + 'infrastructure/docker/services/router/etc/ssl/certs/'):
```

Comment:
We need to be sure that the directory exists, and if not, to create it

Reply:
I added a .gitignore to the certs directory to make sure it exists but I can add this verification if you'd like it